iron-fit-behavior.html

```diff
@@ -123,8 +133,18 @@
       },
 
       /**
-       * The same as setting margin-top and margin-bottom css properties.
-       * @deprecated
```

Comment:
Uhhhhh doesn't this then count as a breaking change? Like pre-this PR, if I am using these properties (but I know I shouldn't), they have behaviour X. After this PR, you're re-using them and they have behaviour Y. Am I misunderstanding?